GCDWebServer/Core/GCDWebServerPrivate.h

```diff
@@ -92,6 +92,13 @@
 
 #define __GCDWEBSERVER_LOGGING_FACILITY_COCOALUMBERJACK__
 
+/**
+ * Whether async should be used by log messages, excluding error messages that are always sent sync.
+ **/
+#ifndef LOG_ASYNC_ENABLED
+    #define LOG_ASYNC_ENABLED YES
```

Comment:
This is related to CocoaLumberjacks change to swift 3.0
I've got an error without this, so I had to add it here in this branch